mantidimaging/gui/windows/operations/presenter.py

```diff
@@ -196,8 +197,22 @@ def _post_filter(self, updated_stacks, task):
             self.view.clear_notification_dialog()
             self.view.show_operation_completed(self.model.selected_filter.filter_name)
 
-    def _do_apply_filter(self, apply_to):
-        self.model.do_apply_filter(apply_to, partial(self._post_filter, apply_to))
+    def _do_apply_filter(self, apply_to, allow_180_degree=False, confirm_with_user_for_180degree=False):
+        confirmed_stacks = []
+        for stack in apply_to:
+            is_180_proj = self.is_a_proj180deg(stack)
+            if not is_180_proj:
+                confirmed_stacks.append(stack)
+                continue
+            if is_180_proj and allow_180_degree:
+                confirmed_stacks.append(stack)
+            else:
+                if is_180_proj and confirm_with_user_for_180degree and\
+                        self.view.ask_confirmation("Are you sure you want to apply to 180 degree projection?"):
```

Comment:
```suggestion
                        self.view.ask_confirmation("Operations applied to the sample are also automatically applied to the 180 degree projection. Please avoid applying an operation unless you're absolutely certain you need to.\nAre you sure you want to apply to 180 degree projection?"):
```

We need to find a way to signify that the users shouldn't be operating on the 180 degree projection (unless there's a reason to, and I'm not sure what that reason might be)

Maybe a better warning message would be enough?

Reply:
Sounds like a plan!